angular.json

```diff
@@ -47,14 +47,6 @@
                   "with": "src/environments/environment.prod.ts"
                 }
               ]
-            },
```

Comment:
Please don't modify this file, just add the key to `environment.ts`. I don't want to add the `environment.dev.ts` file because that will add it to the git history if I change the server address or add other keys and forget to unstage it.

Reply:
Of course, I don't wanna add this file neither. For if I just git clone the project and use `npm start` can't I start the project. So never put an `environment.dev.ts ` and always make our public key public.